client/tests/integration/smartcontracts/deny_new_validators_validator/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deny_new_validators_validator"
-version = "2.0.0-pre-rc.4"
+version = "2.0.0-pre-rc.13"
```

Comment:
So, the way this is updated, is a find-and-replace which searches for the previous full version and replaces it with the next full version. Given that these things should have an independent semver, I propose treating them accordingly, and give them a version number starting with `0.1.0` and bump the versions accordingly (independent of our release cycle). 

This may not be the best solution, however. Everything being tied to the same semver screams "don't even try to use this in isolation from the other tools". So I don't mind it being this way.

Reply:
Yeah, it highly depends on our `iroha_wasm` crate which it-self depends from `iroha_data_model`.
So I think it's okay to have the same semver